.github/workflows/publish-docker-images.yaml

```diff
@@ -0,0 +1,73 @@
+name: Publish Docker Image
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        type: string
+        required: false
+        description: 'The environment to publish the Docker image to.'
+      tags:
+        type: string
+        required: true
+        description: 'The tags to apply to the Docker image.'
+      images:
+        type: string
+        required: true
+        description: 'The images to publish'
+  workflow_dispatch:
```

Comment:
Does the `workflow_dispatch:` need the inputs definition again ?

Reply:
Nope, it extracts the values if they're not set, you can see `images` and `tags` below. The reason i have it like this is so users can't enter `latest` tag on their custom branch which sends everybody's VET to them haha.

So it will take whatever branch you pick and then set the tag to `<branch-name>-<commit-sha>` and it only publishes to Github Packages